pkg/kubelet/metrics/metrics.go

```diff
@@ -28,13 +28,6 @@ import (
 const kubeletSubsystem = "kubelet"
 
 var (
-	ImagePullLatency = prometheus.NewSummary(
```

Comment:
Not with this metric. We have metrics for all Docker operations which would take precedence over this. Although now that you mention it, we'd need to do the same for rkt...Hmm it would be better to have a single metric for it, but the implementation may be harder than it's worth for now. Let me know if you have any ideas.